airbyte-config/models/src/main/resources/types/StandardSourceDefinition.yaml

```diff
@@ -10,7 +10,7 @@ required:
   - dockerRepository
   - dockerImageTag
   - documentationUrl
-additionalProperties: false
+additionalProperties: true
```

Comment:
This may be needed to make the file-based migration system forward compatible. I will test it locally if this is necessary.